src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanIntelAligner.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.utils.smithwaterman;
+
+
+import com.intel.gkl.smithwaterman.IntelSmithWaterman;
+import org.broadinstitute.gatk.nativebindings.smithwaterman.SWOverhangStrategy;
+import org.broadinstitute.gatk.nativebindings.smithwaterman.SWParameters;
+import org.broadinstitute.gatk.nativebindings.smithwaterman.SWAlignerNativeBinding;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+
+public final class SmithWatermanIntelAligner implements SmithWatermanAligner {
+
+
+    /**
+     * return the stateless singleton instance of SmithWatermanIntelAligner
+     */
+
+
+    private final SWAlignerNativeBinding SmithWaterman = new IntelSmithWaterman();
```

Comment:
this variable should start with a lowercase S.  Only types and constants should start with a capital letter.